pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -103,3 +103,8 @@ def configure_progress_bar(self, refresh_rate=1, process_position=0):
 
     def _trainer_has_checkpoint_callbacks(self):
         return len(self.trainer.checkpoint_callbacks) > 0
+
+    def attach_model_logging_functions(self, model):
+        for callback in self.trainer.callbacks:
+            callback.log = model.log
```

Comment:
Just a question. This will still use the stage set within the model for logging, right?

Reply:
Yes, we are just attaching the log functions from the model to each callbacks. 

Right now, users need to do
```
pl_module.log(...)
```

But it makes more sense to support 

```
self.log(...)
```

Reply:
would it be cleaner to have `callback.model_log = model.log`

Reply:
Hey @Borda. The user doesn't need to know the function is coming from the model. 
To the user, he just need to do `self.log` and it works :)